File: learn/11python/01/demo01.py
def demo01():
    i1 = 1
    i2 = 2
    i3 = 1.1
    print(i1 + i2 + i3)
    print("*********************************************")


def demo02():
    s1 = "s1"
    s2 = "s2"
    s3 = s1+s2

    for i in s3:
        print(i)
    print("*********************************************")

def demo03():
    l=[1,2,3,"zz","kx"]
    l.append("append01")
    for i in l:
        print(i)
    print(len(l))
    print("*********************************************")

demo01()
demo02()
demo03()

File: learn/15project/38typescript/demo01.ts
interface Demo {
    Eat(type: string)

    Name: string
}


export class Test {
    firstname: String
    age: number

    public constructor(f: string, s: string, age: number) {
        this.firstname = f + s
        this.age = age
    }

    public demo01() {
    }

    protected demo02() {
    }

    private demo03() {
    }
}


var test = new Test("zz", "kx", 18);
test.demo01()

export function greeter(person) {
    return "Hello, " + person;
}

let user = "Jane User";

document.body.textContent = greeter(user);

function greeter2(person2: number[]) {

}

let user2 = [0, 1, 2];

greeter2(user2);

